drivers/npmx/npmx_shell.c

```diff
@@ -2719,6 +2971,44 @@ static int cmd_reset(const struct shell *shell, size_t argc, char **argv)
 	return 0;
 }
 
+static int ship_mode_set(const struct shell *shell, npmx_ship_task_t ship_task)
```

Comment:
This function and `cmd_ship_mode_ship_set/get` should be above `cmd_reset`.

Reply:
Done